backend/src/backend.rs

```diff
@@ -21,7 +14,12 @@ pub fn __get_blocktime() -> u64 {
 
 #[no_mangle]
 pub fn __caller() -> OdraAddress {
-    casper_env::caller().into()
+    OdraAddress::try_from(casper_env::caller()).unwrap()
+}
+
+#[no_mangle]
+pub fn __self_address() -> OdraAddress {
```

Comment:
Remember to add the corresponding definition in odra/external_api

Reply:
I made a not in https://github.com/odradev/odra/issues/2